testapp/build.gradle

```diff
@@ -35,7 +35,7 @@ android {
     def appName = "Mini App Sample"
     def stagingManifestPlaceholders = [
             baseUrl                 : property("MINIAPP_SERVER_BASE_URL") ?: "https://www.example.com/",
-            isTestMode              : property("IS_TEST_MODE") ?: false,
+            isPreviewMode           : property("IS_PREVIEW_MODE") ?: false,
```

Comment:
This should be `true` by default for the demo app.

Reply:
Ah, yes I missed this spec.